backend/apis/parse-docket-pdf.utils.js

```diff
@@ -1,13 +1,20 @@
-const sectionHeaderRegex = /^([A-Z][A-Z ]+)/;
+const sectionHeaderRegex = /^([A-Z][A-Z ]+)/g;
```

Comment:
Why is the global flag needed for this regex? We're applying it just to one line at a time, right? And only care about the first match.

Reply:
The global flag gives access to the lastIndex property. This could prove useful for many lines that satisfy the regex but hold additional info. Some lines look like this: 
TRUST TOTALS Trust Due: 1,050.00
The TRUST TOTALS matches the regex but we'd want to slice what comes next and put that somewhere. So the lastIndex property could be useful for that kind of case.

Reply:
You're not storing the lastIndex property in the section objects, though. Additionally, the `TRUST TOTALS` line is not.a section header and does not match this regular expression. Note how this regular expression requires the first letter of the line to be a capital letter, but that the TRUST TOTALS line in [this example docket](https://drive.google.com/drive/u/0/folders/1cW6RM4jLDeiLeSy7emI8_xdf6HhpfBhG) has space characters at the beginning of the line.

`sectionHeaderRegex` is meant to find the top-level sections (like `ACCOUNT SUMMARY`). It is not meant to parse information out of lines within those top level sections (like `TRUST TOTALS`).